### PR TITLE
Error handling - access denied

### PR DESCRIPTION
**Pull request by olerichard**
new componenet ResourceErrorMessage.
Takes in a ResourceError and uses data supplied to write out there error.
Has a custom build access denied errorMessage component that also write out the accessRequirements.
If you need to supply own title etc one should still use <ErrorMessage/>.

ResourceErrorMessage implemented for contractOverview,ActiveRequestsPage and ManagePersonnelPage

ResourceError accessRequirements spellingerror fixed.

Changed errorMessage for contractTable in contractOverview to show a noData error.
Also added a custom error message here.

### CODE REVIEW

**Comment by eskil0312** [comment]
Maybe add ResourceErrorMessage to Provisioning and Completed request as well?

File: src/frontend/src/components/ResourceErrorMessage.tsx/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import { ErrorMessage, BlockIcon } from '@equinor/fusion-components';
+import * as styles from './styles.less';
+import ResourceError from '../../reducers/ResourceError';
+import { ErrorTypes } from '@equinor/fusion-components/dist/components/general/ErrorMessage';
+
+const iconProps = {
+    width: 80,
+    height: 80,
+    cursor: 'default',
+    color: '#666666',
```
[the remark this answers is not in the document]
Comment (eskil0312) [reply src/frontend/src/components/ResourceErrorMessage.tsx/index.tsx:11]: and grid unit for width and height

[the remark this answers is not in the document]
Comment (olerichard) [reply src/frontend/src/components/ResourceErrorMessage.tsx/index.tsx:13]: reworked the css to be more like a normal errorMessage. Pretty much copy paste. 

The IconProps remains the same. As this is also a copy from errorMessage component. 
Keeping it this way for consistency between errors.

**Comment by MartinForre** [comment]
@olerichard is the component folder named `ResourceErrorMessage.tsx`?

**Comment by olerichard** [comment]
@MartinForre  yepp, but not anymore

**Pull request merged**